pkg/blaker/errors.go

```diff
@@ -0,0 +1,27 @@
+package blaker
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type SkipError struct {
```

Comment:
better to name BreakError?

Reply:
I think so too. 
fixed https://github.com/cynipe/blaker/pull/3/commits/48f3dd8b0c6a337d204ac6d7a29fa02fa3b53bc1